library/src/main/scala/scala/scalajs/runtime/EnvironmentInfo.scala

```diff
@@ -49,4 +49,20 @@ trait EnvironmentInfo extends js.Object {
   def sourceMapper: js.UndefOr[js.Function1[
     js.Array[JSStackTraceElem], js.Array[JSStackTraceElem]]] = js.native
 
+  /** Semantics configuration.
+   *
+   *  @group envInfo
+   */
+  val semantics: EnvironmentInfo.Semantics = js.native
```

Comment:
The environment info is not the right place to put this information. It contains information about the **environment** Scala.js runs in, not static information about a given linked "binary".

I propose a similar `LinkingInfo` which has the additional restriction, that all its members must (or should) be intrinsics.

Note the difference: A given linked Scala.js program works with any (valid) `EnvironmentInfo`. However, it will only work properly with a single `LinkingInfo`.